terraform/environments/bootstrap/delegate-access/policies.tf

```diff
@@ -365,3 +365,129 @@ data "aws_iam_policy_document" "migration_additional" {
     resources = ["*"] #tfsec:ignore:AWS099 tfsec:ignore:AWS097
   }
 }
+
+
+# developer policy - member SSO and collaborators
+resource "aws_iam_policy" "database-mgmt" {
+  provider = aws.workspace
+  name     = "database_mgmt_policy"
+  path     = "/"
+  policy   = data.aws_iam_policy_document.database-mgmt-document.json
+}
+
+
+#tfsec:ignore:aws-iam-no-policy-wildcards
+data "aws_iam_policy_document" "database-mgmt-document" {
+  #checkov:skip=CKV_AWS_108
+  #checkov:skip=CKV_AWS_109
+  #checkov:skip=CKV_AWS_111
+  #checkov:skip=CKV_AWS_110
+  source_policy_documents = [data.aws_iam_policy_document.common_statements.json]
+  statement {
+    sid    = "databaseAllow"
+    effect = "Allow"
+    actions = [
+      "acm:ImportCertificate",
+      "autoscaling:UpdateAutoScalingGroup",
+      "aws-marketplace:ViewSubscriptions",
+      "cloudwatch:PutDashboard",
+      "cloudwatch:ListMetrics",
+      "cloudwatch:DeleteDashboards",
+      "codebuild:ImportSourceCredentials",
+      "codebuild:PersistOAuthToken",
+      "ds:*Tags*",
+      "ds:*Snapshot*",
+      "ec2:StartInstances",
+      "ec2:StopInstances",
+      "ec2:RebootInstances",
+      "ec2:ModifyImageAttribute",
+      "ec2:ModifySnapshotAttribute",
+      "ec2:CopyImage",
+      "ec2:CreateImage",
+      "ec2:CopySnapshot",
+      "ec2:CreateSnapshot",
+      "ec2:CreateSnapshots",
+      "ec2:CreateTags",
+      "ec2:DescribeVolumes",
+      "ec2:DescribeInstances",
+      "ec2:DescribeInstanceTypes",
+      "ecs:StartTask",
+      "ecs:StopTask",
+      "identitystore:DescribeUser",
+      "kms:Decrypt*",
+      "kms:Encrypt",
+      "kms:ReEncrypt*",
+      "kms:GenerateDataKey*",
+      "kms:DescribeKey",
+      "lambda:InvokeFunction",
+      "lambda:UpdateFunctionCode",
+      "rds:CopyDBSnapshot",
+      "rds:CopyDBClusterSnapshot",
+      "rds:CreateDBSnapshot",
+      "rds:CreateDBClusterSnapshot",
+      "rds:RebootDB*",
+      "rhelkb:GetRhelURL",
+      "s3:PutObject",
+      "s3:DeleteObject",
+      "secretsmanager:GetResourcePolicy",
+      "secretsmanager:GetSecretValue",
+      "secretsmanager:DescribeSecret",
+      "secretsmanager:ListSecretVersionIds",
+      "secretsmanager:PutSecretValue",
+      "secretsmanager:UpdateSecret",
+      "secretsmanager:RestoreSecret",
+      "secretsmanager:RotateSecret",
+      "ssm:*",
+      "ssm-guiconnect:*",
+      "sso:ListDirectoryAssociations",
+      "support:*"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    sid    = "snsAllow"
+    effect = "Allow"
+    actions = [
+      "sns:Publish"
+    ]
+    resources = ["arn:aws:sns:*:*:Automation*"]
+  }
+
+  statement {
+    sid    = "lambdaAllow"
+    effect = "Allow"
+    actions = [
+      "lambda:InvokeFunction"
+    ]
+    resources = ["arn:aws:lambda:*:*:function:Automation*"]
+  }
+
+  statement {
```

Comment:
I think this statement should be removed as it's probably too excessive for this role